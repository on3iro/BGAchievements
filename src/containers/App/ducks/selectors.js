export const getLocation = state => state.router.location
