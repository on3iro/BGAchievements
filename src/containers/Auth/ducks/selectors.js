export const getLoggedIn = state => state.auth.user.isLoggedIn
export const getRegistered = state => state.auth.user.hasRegistered
export const getMaxAccounts = state => state.auth.user.data.maxAccounts
export const getUserID = state => state.auth.user.data.id
export const getAuthToken = state => state.auth.user.token
export const getLoadingState = state => state.auth.AuthRequestState.isLoading
