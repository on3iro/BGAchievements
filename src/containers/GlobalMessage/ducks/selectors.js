export const getError = state => state.GlobalMessages.GlobalError
export const getMessage = state => state.GlobalMessages.GlobalMessage
