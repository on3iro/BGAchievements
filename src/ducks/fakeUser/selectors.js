// @flow

import type { State } from 'globalTypes'

import type { User } from './types'

export const getFakeUser = (state: State): ?User => state.fakeUser.user
export const getUserLoading = (state: State): boolean => state.fakeUser.loading
