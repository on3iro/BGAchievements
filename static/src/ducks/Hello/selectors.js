export const getSampleData = (state) => state.Hello.sampleData
export const getLoading = (state) => state.Hello.loading
