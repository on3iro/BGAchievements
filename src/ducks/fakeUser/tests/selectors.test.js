import * as selectors from '../selectors'

describe('FakeUser selectors', () => {
  it('Should get fake user', () => {
    expect(selectors.getFakeUser({
      fakeUser: {
        user: {
          id: 'some data'
        }
      }
    })).toMatchSnapshot()
  })

  it('Should get user loading state', () => {
    expect(selectors.getUserLoading({
      fakeUser: {
        loading: true
      }
    })).toMatchSnapshot()
  })
}) // end FakeUser selectors
