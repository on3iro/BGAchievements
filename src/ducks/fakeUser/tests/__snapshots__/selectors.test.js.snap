// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FakeUser selectors Should get fake user 1`] = `
Object {
  "id": "some data",
}
`;

exports[`FakeUser selectors Should get user loading state 1`] = `true`;
